### PR TITLE
lntest/channels: introduce subpackage to deduplicate static structs

### PR DESCRIPTION
**Pull request by Crypt-iQ**
Deduplicates `testTx`, `testHdSeed`, and private keys in several places. Can't yet remove the one from `lnwallet/test_utils.go` and use `lntest/channels` as it leads to an import cycle.

### CODE REVIEW

File: lntest/channels/channels.go [hunk 1]
```diff
@@ -1,35 +1,37 @@
...
 		0xb7, 0x94, 0x38, 0x5f, 0x2d, 0x1e, 0xf7, 0xab,
 		0x4d, 0x92, 0x73, 0xd1, 0x90, 0x63, 0x81, 0xb4,
 		0x4f, 0x2f, 0x6f, 0x25, 0x88, 0xa3, 0xef, 0xb9,
 		0x6a, 0x49, 0x18, 0x83, 0x31, 0x98, 0x47, 0x53,
 	}
 
-	// testTx is used as the default funding txn for single-funder channels.
-	testTx = &wire.MsgTx{
+	// TestTx is a serializable txn for testing funding txn.
+	TestTx = &wire.MsgTx{
```
Comment (cfromknecht) [inline lntest/channels/channels.go:34]: nit: rename to `TestFundingTx` to clarify?

**Review by cfromknecht — APPROVED** [review]
LGTM, just one nit 👍

**Review by bhandras — APPROVED** [review]
LGTM :1st_place_medal:

**Comment by Crypt-iQ** [comment]
Should the subpackage be renamed from `channels` to something more descriptive/indicative of the contents?

**Comment by bhandras** [comment]
> Should the subpackage be renamed from `channels` to something more descriptive/indicative of the contents?

Imo `lntest/channels` is good enough.

**Review by bhandras — APPROVED** [review]

**Review by cfromknecht — APPROVED** [review]
LGTM 🎉

**Pull request merged**